core/minorblockchain.go

```diff
@@ -282,9 +280,9 @@ func (m *MinorBlockChain) loadLastState() error {
 	if _, err := m.StateAt(currentBlock.GetMetaData().Root); err != nil {
 		// Dangling block without a state associated, init from scratch
 		log.Warn("Head state missing, repairing chain", "number", currentBlock.NumberU64(), "hash", currentBlock.Hash())
-		if err := m.repair(&currentBlock); err != nil {
-			return err
-		}
+		//if err := m.repair(&currentBlock); err != nil {
```

Comment:
why remove?

Reply:
will repair in InitFromRootBlock
will remove repair func